### PR TITLE
Fix to update hasNextPage, was always false before

### PR DESCRIPTION
**Pull request by noodlespace**
hasNextPage was always false not matter what the filter criteria were, pushed through list_length to fix this from get_query

### CODE REVIEW

**Comment by abawchen** [comment]
@noodlespace : Thanks for this PR! :smiley:

But the test case failed ..., would you please take a look? https://travis-ci.org/graphql-python/graphene-mongo/builds/370920862?utm_source=github_status&utm_medium=notification

```
graphene_mongo/fields.py:165:1: W293 blank line contains whitespace
graphene_mongo/fields.py:166:26: F821 undefined name 'query'
graphene_mongo/fields.py:167:28: F821 undefined name 'query'
graphene_mongo/fields.py:168:24: F821 undefined name 'query'
graphene_mongo/fields.py:170:28: F821 undefined name 'query'
make: *** [lint] Error 1
```

- The first one is checking by flake8
- The second one seems like a syntax error

any discussion is welcome.

File: graphene_mongo/fields.py [hunk 1]
```diff
@@ -138,14 +138,16 @@ def get_query(cls, model, info, **args):
...
+            list_length = objs.count()
 
             if first is not None:
                 objs = objs[:first]
             if last is not None:
                 # https://github.com/graphql-python/graphene-mongo/issues/20
                 objs = objs[-(last+1):]
 
-        return objs
+        return {'objs': objs, 'length': list_length}
```
Comment (abawchen) [inline graphene_mongo/fields.py]: I think we can:
```python
return objs, length
```
and

```
return [], 0
```
at line 107

File: graphene_mongo/fields.py [hunk 2]
```diff
@@ -160,7 +162,15 @@ def connection_resolver(cls, resolver, connection, model, root, info, **args):
         iterable = resolver(root, info, **args)
         if not iterable:
             iterable = cls.get_query(model, info, **args)
-        _len = len(iterable)
```
Comment (abawchen) [inline graphene_mongo/fields.py]: And then we can simplify as follows to get `iterable` and `_len`:
```python
iterable, _len = cls.get_query(model, info, **args)
```

Comment (noodlespace) [reply graphene_mongo/fields.py]: I'll have a look at it today
Hopefully I will be more active on this account now

**Comment by noodlespace** [comment]
@abawchen I have made the requested updates, not quite sure to do with the coverall tests though

**Comment by abawchen** [comment]
@noodlespace : Thanks for the efforts! Merged 👍

**Pull request merged**